src/utils/index.ts

```diff
@@ -0,0 +1,26 @@
+import { EventFormPublicType, Hall, SignedInUser, SignedOutUser } from '@app/types';
+
+export const canUpdateEvent = (
```

Comment:
This seems like a very event specific thing to put into the index file of a utils directory. This should be moved into the events subdirectory

Reply:
sorry, which events subdirectory?

Reply:
ended up moving it to Event.tsx and deleting this file - https://github.com/resident51/resident51/pull/6/commits/d7baad4b650baf40e7d872b957007b5de85e963b